### ISSUE TITLE
Add a config to skip asteroid generation

### ISSUE THREAD

**Issue by 9600bauds** [d3athrow/vgstation13#10177]
to speed up listen serb startup when you don't need to do anything at the asteroid

**Comment by clusterfack** [comment]
No, just make a test map. We need a test map. Someone make a test map.

**Comment by Exxion** [comment]
Vaults too

**Comment by 9600bauds** [comment]
But clusterfack
What if I need to test something with the normal map

**Comment by PJB3005** [comment]
@Exxion you can already disable vault gen.

**Comment by Exxion** [comment]
Really? I looked for a config option but didn't see it.
Guess I didn't look hard enough.

**Comment by clusterfack** [comment]
I don't know bauds, why don't you compile the normal map then?

**Comment by 9600bauds** [comment]
But then I compile in the normal map and it still takes me like two to four more seconds to test it because I still can't disable asteroid generation for some reason.
Please, I was going to waste those four seconds of my life elsewhere.

**Comment by Probe1** [comment]
:laughing: 

we're arguing over who won't code this minor tester quality of life config line

https://youtu.be/pwO2r0-WU9E?t=50

**Comment by clusterfack** [comment]
Probe make us a test map you fag

**Comment by Probe1** [comment]
alright

**Comment by Probe1** [comment]
I regret posting now.

**Comment by xpcybic** [comment]
I guess I could make a PR for what I did with Heartstation so far 
It's like 180x200x2 and no asteroid/vaults, it loads super fast but it's still semi-shitty and probably lacks a lot of modern amenities

**Comment by west3436** [comment]
10 year old feature request
